app/components/viral/form/prefixed/boolean_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Viral
+  module Form
+    module Prefixed
+      # Render a truthy radio component
+      class BooleanComponent < Viral::Component
+        attr_reader :form, :name, :value
+
+        renders_one :prefix
```

Comment:
**question:** Why did you decide to use `renders_one :prefix` instead of just having the `prefix` passed in as a parameter or just use the `name` as the prefix?